### ISSUE TITLE
Live Region not speaking in JAWS 2018

### ISSUE THREAD

**Issue by stevefaulkner** [FreedomScientific/VFO-standards-support#40]
Chrome: 65.0.3322.3 Canary 64-bit
Chrome: 63.0.3239.132 (Official Build) (64-bit)
Chrome: 55.0.2841.0 (Official Build) (64-bit)
Firefox: 52
JAWS: 2018
NVDA: 2017.4

**Steps to repro:**

1. With JAWS running, visit http://ncamftp.wgbh.org/sp/tests/simpleLiveRegion/simpleLiveRegionDemo.html
2. Press the "Add One and Display" button multiple times
3. Notice that text is added, but JAWS does not speak it
4. Try the same with NVDA
5. Notice that NVDA speaks the added text
6. Try JAWS in Firefox
7. Notice that JAWS speaks the added text

**Expected:** JAWS should read added text in Chrome

**Actual:** JAWS does not read added text.
 
original bug report: https://bugs.chromium.org/p/chromium/issues/detail?id=802337

**Comment by stevefaulkner** [comment]
added internally as Bug 98808 - Live Region not speaking in JAWS 2018

**Comment by stevefaulkner** [comment]
A similar  bug with JAWS 2018, occurs in chrome only (works with IE 11 and Firefox). NVDA it works fine. 
It also works fine with JAWS 18.0.453 + chrome so is a regression

content of div with aria-atomic=true not announced when updated, but is announced when aria-atomic on child element:
**content update not announced:**
`<div aria-live="polite" aria-atomic="true" id="messageTextAnnounce"></div>`

**content update announced:**
`<div aria-live="polite"><span id="messageNodeAnnounce" aria-atomic="true"></span></div>`

test case and notes: https://s.codepen.io/stevef/debug/JpjOdY

**Comment by stevefaulkner** [comment]
Looks like it has been fixed and will be heading down the pipe in next point release of JAWS 2018
